src/main/java/com/plaid/client/response/PaymentRecipientListResponse.java

```diff
@@ -0,0 +1,15 @@
+package com.plaid.client.response;
+
+import com.plaid.client.model.payment.Address;
+import com.plaid.client.model.payment.Recipient;
+import com.plaid.client.response.PaymentRecipientGetResponse;
```

Comment:
Doesn't look like this import is currently used

Reply:
⚡️opening a new PR to address this